CovertInfiltration/Src/CovertInfiltration/Classes/UISSManager_CovertAction.uc

```diff
@@ -218,4 +232,13 @@ simulated protected function UnsubscribeFromAllEvents()
 
     ThisObj = self;
     `XEVENTMGR.UnRegisterFromAllEvents(ThisObj);
+}
+
+///////////////
+/// Helpers ///
+///////////////
+
+simulated function XComGameState_CovertAction GetAction()
+{
+	return CovertOpsSrceen.GetAction();
```

Comment:
`return CovertOpsSrceen.GetAction();`

I don't know if this typo (on line 243 in UISSManager_CovertAction) will actually break anything, but it's probably a good idea to fix it regardless.

Reply:
No, because the declaration is also messed up :P

```
var UICovertActionsGeoscape CovertOpsSrceen;
```